pkg/bot/mattermost.go

```diff
@@ -219,28 +269,15 @@ func (b MMBot) getTeam() *model.Team {
 }
 
 // Check if BotKube user exists in Mattermost
-func (b MMBot) getUser() *model.User {
+func (b *Mattermost) getUser() *model.User {
 	users, resp := b.APIClient.AutocompleteUsersInTeam(b.getTeam().Id, b.BotName, 1, "")
 	if resp.Error != nil {
 		b.log.Fatalf("There was a problem finding Mattermost user %s. %s", b.BotName, resp.Error)
 	}
 	return users.Users[0]
 }
 
-// Create channel if not present and add BotKube user in channel
```

Comment:
Now I understand why got rid of the `ChannelName` and started working with the `ChannelID`.

Reply:
What do you mean? 🤔 I removed this part because:
- the Mattermost instruction tells to invite the bot manually
- the error wasn't checked and I'm not sure if the bot can invite itself to a channel. I guess it didn't work properly

So let's unify and simplify that 🙂